frb_codegen/src/parser/ty.rs

```diff
@@ -331,18 +333,21 @@ impl<'a> TypeParser<'a> {
                                 .get(&ident_string)
                                 .map(IrStruct::is_empty)
                                 .unwrap_or(false),
+                            is_exception: false,
                         }))
                     }
 
-                    [(name, None)] if self.src_enums.contains_key(&name.to_string()) => {
+                    [(name, _)] if self.src_enums.contains_key(&name.to_string()) => {
                         let ident_string = name.to_string();
                         if self.parsed_enums.insert(ident_string.to_owned()) {
+                            // NOTE: not working
```

Comment:
👀 what is happening

Reply:
https://github.com/fzyzcjy/flutter_rust_bridge/pull/1325/commits/83a78894084d0e81fd0f40647b125d745a5d0adc#diff-cf4e6088e3742e99ecc674bc551f63661342a4550589d29f5a387eddc772e09d
this part should be fixed in this commit